dev/script/generate-config/.nvmrc

```diff
@@ -1,2 +1,2 @@
-8
+15
```

Comment:
I'd generally suggest sticking with even number releases as they are supported for far longer than the odd releases. Unless there is a good reason to require 15.